src/util/options.schema.json

```diff
@@ -1357,7 +1357,10 @@
           "description":
             "Lists of activated transformations. Transformations happen after analyses.",
           "type": "array",
-          "items": { "type": "string" },
+          "items": {
+            "type": "string",
+            "enum": ["partial", "expeval", "assert", "remove_dead_code"]
```

Comment:
Constrain ordering: remove dead code before adding assertions.

Reply:
Upon testing, this almost certainly needs to be the other way around, i.e. `assert` before `remove_dead_code`. Otherwise, assertions added by `assert` may reference variables that have already been removed.

And the initial problem as described above does not exist: assertions are not inserted around dead code, by virtue of that code being dead.